### ISSUE TITLE
Cancel a sell order

### ISSUE THREAD

**Issue by NicolasJEngler** [ProjectOpenSea/opensea-js#131]
Does the SDK offer a method to cancel a sell order/listing?

**Comment by hqtoan94** [comment]
+1

**Comment by hqtoan94** [comment]
They don't have it yet in the document but inside the codebase, they do have cancelOrder method.
https://github.com/ProjectOpenSea/opensea-js/blob/7949ef7ca50d23e1a766af5af85e99d6e057d7a4/src/seaport.ts#L884

**Comment by NicolasJEngler** [comment]
Yes, even though the team ended up finding that available method, we moved away from this feature. I'll close the issue, but it wouldn't hurt to add this to the current docs.
